recipe/meta.yaml

```diff
@@ -19,33 +20,54 @@ requirements:
     - wheel
     - pip
   run:
-    - python >=3.6
+    - python
     - jupyter_client >=4.2
     - jupyter_core
+    # nb_conda_kernels expexts jupyter_server or notebook <7 at runtime,
+    # see https://github.com/anaconda/nb_conda_kernels/blob/2.5.0/nb_conda_kernels/install.py#L16-L29
+    # but jupyter_server isn't mentioned in requirements
+    - jupyter_server
```

Comment:
Please don't include this. We need _either_ `jupyter_server` or `notebook`, and it's important that we don't require either, so that this package remains compatible with classic notebook environments

Reply:
Since this package has no function without Jupyter Notebook or JupyterLab, we can safely assume that one of these requirements is satisfied.